app.js

```diff
@@ -94,22 +94,46 @@ function checkFinger(img, prediction, numFingers) {
   const landmarks = normalizePredictions(prediction);
   console.log('Landmarks for model:')
   console.log(landmarks);
-  // Pass these landmarks to the model
-  return true;
+  let correct = false
+  // Pass these landmarks to the classifier
+  return fetch('http://127.0.0.1:8000', {
+    method: 'POST', 
+    body: JSON.stringify(landmarks)
+  }).then(
+      response => response.text()
+  ).then(
+      body => {
+          console.log(body)
+          correct = (parseInt(body) === numFingers)
+          if (correct) {
+              console.log("You're correct!")
+          } else {
+              console.log("Incorrect. Expected: " + numFingers + "; Actual: " + body)
+          }
+          return correct
+      }
+  )
 }
 
 function checkQuadrantAndFinger(img, prediction, target) {
-  const quadrantCorrect = checkQuadrant(img, prediction.boundingBox, target.quadrant);
-  return quadrantCorrect && checkFinger(img, prediction, target.numFingers);    
+    if (!prediction) {
+        return false
+    }
+    const quadrantCorrect = checkQuadrant(img, prediction.boundingBox, target.quadrant);
+    return checkFinger(img, prediction, target.numFingers).then(
+        correct => {
+            return correct && quadrantCorrect
+        }
+    );    
```

Comment:
we want to do `quadrantCorrect && something` so the api call doesn't happen if the quadrant is already wrong. i think this should be
```
return quadrantCorrect && checkFinger(img, prediction, target.numFingers)
```
which i now see is what i originally had, but i think you fixed the promise issue in `checkImage`.
can you change this and test if it still works?

Reply:
Wooo it still works!